fsh/terminology/supportFactor.fsh

```diff
@@ -22,4 +31,28 @@ Description: "The set of all possible support factors values (not specific to an
 * ^url = "https://standards.digital.health.nz/fhir/ValueSet/sact-support-factor-value-code"
 * ^jurisdiction.coding = urn:iso:std:iso:3166#NZ
 
-* include codes from system $SNOMED
```

Comment:
I presume this meant that any snomed code was valid?  I have removed this as we now have a discrete list.